### DISCUSSION TITLE
Why migration files are nested in directories ?

### DISCUSSION THREAD

**Discussion by felicienfrancois** [prisma/prisma#19560] [Q&A] [answered]
```
migrations/
  └─ 20210313140442_init/
    └─ migration.sql
  └─ 20210313140442_added_job_title/
    └─ migration.sql
```

Is there any specific reason for not using a flatter pattern, like the one of all other frameworks I experienced :
```
migrations/
  └─ 20210313140442_init.sql
  └─ 20210313140442_added_job_title.sql
```

Which makes migration files searchable by filename and navigation much easier to my opinion.

Is there a way to configure prisma to use the flat pattern ?

**Comment by ludralph** [comment, marked as answer]
Hi @felicienfrancois 👋 

To answer the questions in reverse order:

- This is currently not configurable.
- The nesting exists because we anticipated migrations being composed of multiple files: typescript hooks, maybe docs (README), maybe a down migration.

**Reply by nurul3101** [reply]
@felicienfrancois Did you had a chance to look @ludralph's response?

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
